### PR TITLE
bump to 1.17.0 for next release

### PR DESCRIPTION
**Pull request by laurafitzgerald**
# Issue link

# What

# Verification steps

### CODE REVIEW

File: bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,719 @@
...
+          "metadata": {
+            "name": "example-rhmiconfig"
+          },
+          "spec": {}
+        }
+      ]
+    capabilities: Basic Install
+    categories: Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/integreatly/managed-api-service:rhoam-v1.1.0
```
Comment (laurafitzgerald) [inline bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml:54]: This was generated. Maybe there's some change needed in the automation?

File: PROJECT [hunk 2]
```diff
@@ -1,6 +1,6 @@
 domain: integreatly.org
 layout: go.kubebuilder.io/v2
-projectName: integreatly-operator
+projectName: managed-api-service
```
Comment (MStokluska) [inline PROJECT:3]: hmm :/ I don't think we should be updating this

File: bundle.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+FROM scratch
+
+# Core bundle labels.
```
Comment (MStokluska) [inline bundle.Dockerfile:3]: this file can go?

File: bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml (hunk 1 above)
Comment (MStokluska) [reply bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml:54]: hmm this is worrying. I'll look into it

File: bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,719 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
```
Comment (MStokluska) [inline bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml:6]: containerImages seems to be gone :/

File: bundle.Dockerfile (hunk 3 above)
Comment (laurafitzgerald) [reply bundle.Dockerfile:3]: It seems we have this one for 1.16.0. 🤔

File: bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml (hunk 4 above)
Comment (laurafitzgerald) [reply bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml:6]: Does it normally get added by the prepare-release script? Is there some configuration I'm missing that it didn't generate it?

File: bundle.Dockerfile (hunk 3 above)
Comment (laurafitzgerald) [reply bundle.Dockerfile:3]: I'ver removed it for now anyway. 👍

**Pull request merged**